src/main/java/appeng/helpers/PatternHelper.java

```diff
@@ -85,7 +85,13 @@ public PatternHelper( final ItemStack is, final World w )
 
 		for( int x = 0; x < inTag.tagCount(); x++ )
 		{
-			final ItemStack gs = new ItemStack( inTag.getCompoundTagAt( x ) );
+			NBTTagCompound ingredient = inTag.getCompoundTagAt( x );
+			final ItemStack gs = new ItemStack( ingredient );
+
+			if( !ingredient.hasNoTags() && gs.isEmpty() )
+			{
+				//throw new IllegalArgumentException( "No pattern here!" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)